TabulateSmarterTestPackage/Tabulators/ItemTabulator.cs

```diff
@@ -371,9 +370,9 @@ public ItemTabulator()
                 var j = 0;
                 foreach (var p in performanceLevels)
                 {
-                    itemFields[ItemFieldNamesCount + j++] = p.PerfLevel;
-                    itemFields[ItemFieldNamesCount + j++] = p.ScaledLow;
-                    itemFields[ItemFieldNamesCount + j++] = p.ScaledHigh;
+                    itemFields[ItemFieldNamesCount - 3 * performanceLevels.Count + j++] = p.PerfLevel;
```

Comment:
Can you use parenthesis so the order of operation is clear?

Also, can you explain what the new calculation is for?